operator/internal/manifests/internal/gateway/options.go

```diff
@@ -28,6 +28,5 @@ type Secret struct {
 
 // TenantData defines the existing tenantID and cookieSecret for lokistack reconcile.
 type TenantData struct {
-	TenantID     string
```

Comment:
It looks to me as if `CookieSecret` is also not read from here anymore but directly from `OpenShiftOptions.Authentication`. On the other hand I'm also not sure if `TenantID` is still read somewhere.

Reply:
Oh `CookieSecret` is the only option that gets read in `operator/internal/manifests/gateway.go` specifically when we translate from `TenantConfigs` to `TenantData`. TenantID was obsolete since we switched from UUIDs to names, i.e. `application`, `infrastructure`, `audit`.